bundles/org.openhab.binding.worxlandroid/README.md

```diff
@@ -0,0 +1,649 @@
+<img src="images/landroid.svg">
```

Comment:
Please remove, this logo can be uploaded after the binding is merged, so that it ends up in a proper place the UI expects it.

Reply:
Changed